README.md

```diff
@@ -101,7 +101,35 @@ Click the button below to start deployment.
    **Note:** This value defaults to `$Default`; you can reuse this consumer group if there are no other consumers of this Event Hub. If there are other consumers of the Event Hub, a separate consumer group should be created for the Alert Logic collector, and its name typed here.
 
    **Event Hub Filter Json** - Type the filter in JSON format.
-   **example:** {"resultType":"Success"}
+   **example json log:** log = [
+      {
+         "resultType1":"Success1"
+      },
+      {
+         "resultType2": {
+            "status":"Success2"
+         }
+      },
+      {
+         "resultType3": {
+            "status": {
+               "result":"Success3"
+            },
+            "type": {
+               "value": "result"
+            }
+         }
+      }
+   ]
+   **root level filtering example:** {"resultType":"Success"}, result --> [log[0]]
+   **child level filtering example:** {"resultType2": {"status":"Success2"}}, result --> [log[1]]
+   **deeper child level filtering example:** {"resultType3": {"status": {"result":"Success3}}}, result --> [log[2]]
+   **Note:** Child level filtering can go deep with the proper sequence of the object
+
+   **Event Hub Filter Json (AND/OR Condition)** - AND/OR Condition filtering
+   **AND condition filtering** [{"resultType":"Success"},{"resultType2": {"status":"Success2"}}], result --> [log[0],log[1]]
+   **OR condition filtering** [{"resultType":"Success"},{"someOtherResultType": {"status":"Success"}}], result --> [log[0]]
+   **OR condition for same object filtering** [{"resultType3":{"status": {"result":"Success3"}}},{"resultType3": {"type":{"value":"result"}}}], result --> [log[0]]
```

Comment:
result here is `[log[2]]`

Reply:
I suggest we present it in a form of a table, like
Give the input array of messages as above
|  Filter | Output  |
|---|---|
|  <pre>{<br>  "resultType1":"Success1"<br>}</pre> | <pre>{<br>  "resultType1":"Success1"<br>}</pre> |
|   |   |

Reply:
Also please add more properties into sample messages in order `Filter` and `Output` don't look same.

Reply:
Done